src/control/Control.Attribution.js

```diff
@@ -119,7 +119,7 @@ export var Attribution = Control.extend({
 			prefixAndAttribs.push(attribs.join(', '));
 		}
 
-		this._container.innerHTML = prefixAndAttribs.join(' | ');
+		this._container.innerHTML = prefixAndAttribs.join(' <span aria-hidden="true">|</span> ');

```

Comment:
Attaching a video with audio capture (you may have to unmute the video) showing the problem on Android TalkBack:

https://user-images.githubusercontent.com/26493779/149366955-0d5acf0a-f7f8-4740-91db-b5739d45186c.mp4

`aria-hidden="true"` should prevent TalkBack from "focusing" and announcing the vertical line separator.